lib/logstash/outputs/dynatrace.rb

```diff
@@ -81,6 +83,7 @@ def multi_receive(events)
           failure_message = "Dynatrace returned #{response.code} #{response.message}."
 
           if response.is_a? Net::HTTPServerError
+            @logger.debug("Server returned an error")
```

Comment:
I guess the `failure_message` is included in the exception below, so we don't include it again here in this log?

Reply:
Actually yes, which means this log isn't useful at all. I'll remove it.